### ISSUE TITLE
BUG: key in box always return True when default_box=True

### ISSUE THREAD

**Issue by criver** [cdgriffith/Box#42]
For default box, any key membership test returns `True` even when they are not in box while `has_key` correctly returns False.

```
>>> box = Box(default_box=True)
>>> print('a' in box)
True
>>> print(box.has_key('a'))
False
```

**Comment by criver** [comment]
PR to fix this: https://github.com/criver/Box/pull/1. Please let me know if this is good or not. Will create PR to merge into master.

**Comment by cdgriffith** [comment]
Thanks for this catch @criver ! 

If you could change the PR to just remove the entire function should accomplish the same thing, then go ahead and do a PR to `development` and I'll merge it 😄 

I'll need to add some tests for this before a bug release, because I added it for some reason, but of course now can't remember why, and your fix doesn't break any current tests ¯\_(ツ)_/¯

**Comment by cdgriffith** [comment]
Fixed in 3.1.1 https://github.com/cdgriffith/Box/commit/37ad92655da7dc31d3cd55ffab304ea2a0d68a72
